src/overlays/actors/ovl_En_Slime/z_en_slime.h

```diff
@@ -10,7 +17,32 @@ typedef void (*EnSlimeActionFunc)(struct EnSlime*, PlayState*);
 typedef struct EnSlime {
     /* 0x000 */ Actor actor;
     /* 0x144 */ EnSlimeActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0xC0];
+    /* 0x148 */ u8 iceBlockTimer;
+    /* 0x149 */ u8 eyeTexIndex;
+    /* 0x14A */ u8 damageEffectType;
+    /* 0x14C */ s16 timer;
+    /* 0x14E */ s16 idleRotY;
+    /* 0x150 */ s16 respawnRotY;
+    /* 0x152 */ s16 respawnTime;
+    /* 0x154 */ union { 
+                    Vec3f iceBlockSnapPos;
+                    Vec3f wobbleRot;
+                };
+    /* 0x160 */ TexturePtr dropObjectTex;
+    /* 0x164 */ f32 effectAlpha;
+    /* 0x168 */ f32 effectScale;
+    /* 0x16C */ f32 frozenSteamScale;
```

Comment:
And these are normally called `drawDmgEffAlpha`, `drawDmgEffScale`, and `drawDmgEffFrozenSteamScale`, respectively.

Reply:
A lot of these comments I made previously and were resolved, maybe it they forgot to push?

Reply:
Sorry about that! I was working on it last Friday when I lost power for 18 hours, so my desktop just snapped off. I didn't think I lost anything, but it's possible I made changes that I lost and didn't notice. It's also possible I thought I made changes that I never did - was a bit scatterbrained last weekend. I'll be sure to look over your comments again for the next push.

Reply:
No worries!